eng/dogfood.ps1

```diff
@@ -46,6 +46,9 @@ try {
   $env:PATH = "$TestDotnetRoot;$env:Path"
   $env:DOTNET_ROOT = $TestDotnetRoot
 
+  # Avoid downloading Microsoft.Net.Sdk.Compilers.Toolset from feed
```

Comment:
Can you add a bit about it trying to download the -dev version here so it's clearer exactly why this is disabled here?